library/src/main/java/com/github/windsekirun/naraeimagepicker/module/PickerSet.kt

```diff
@@ -55,29 +58,37 @@ object PickerSet {
             val photoCursor = context.contentResolver.query(cursorUri,
                     arrayOf(ID_COLUMN, PATH_COLUMN, DATE_MODIFIED), "$BUCKET_DISPLAY_NAME =?", arrayOf(title), ORDER_BY)
 
-            photoCursor.doWhile {
+            photoCursor?.doWhile {
                 val image = photoCursor.getColumnString(PATH_COLUMN)
                 val id = photoCursor.getColumnString(ID_COLUMN)
                 val file = image.toFile()
-                if (file.exists()) list.add(FileItem(id, image))
+                if (file.exists())
+                    if (item.includeGif) {
+                        list.add(FileItem(id, image))
+                    } else
+                        if (file.extension != "gif")
+                            list.add(FileItem(id, image))
+
             }
 
+            photoCursor?.close()
             list.reverse()
             addPictureMap(title, list)
         }
 
+        titleCursor?.close()
         titleItemSet.clear()
         titleItemList.clear()
         callback.invoke()
     }
 
-    fun getFolderList(): List<com.github.windsekirun.naraeimagepicker.item.FolderItem> {
+    fun getFolderList(): List<FolderItem> {
         return PICTURE_MAP.entries
                 .asSequence()
                 .filter { it.value.isNotEmpty() }
                 .map { it.key to it.value[0] }
-                .map { com.github.windsekirun.naraeimagepicker.item.FolderItem(it.first, it.second.imagePath) }
-                .sortedBy { it.name.toLowerCase() }
+                .map { FolderItem(it.first, it.second.imagePath) }
+                .sortedBy { it.name.toLowerCase(Locale.ENGLISH) }
```

Comment:
Do you have reason to using 'Locale.ENGLISH'? if not, Use 'Locale.getDefault()' will support other conturies.